app/src/views/OperationalLearning/OperationalLearningMap/index.tsx

```diff
@@ -0,0 +1,254 @@
+    );
+    const learningCount = useMemo(() => (
+        learning?.learning_by_country
+            .filter((country) => country.count > 0)
+    ), [learning?.learning_by_country]);
+
+    const maxScaleValue = useMemo(() => (
+        Math.max(
+            ...(learningCount
+                ?.map((activity: { count: number; }) => activity.count)
+                .filter(isDefined) ?? []),
+        )), [learningCount]);
+
+    const {
+        bluePointHaloCirclePaint,
+    } = useMemo(
+        () => ({
+            bluePointHaloCirclePaint: getPointCircleHaloPaint(COLOR_BLUE, 'operation_count', maxScaleValue),
+        }),
+        [maxScaleValue],
+    );
+
+    const handleCountryClick = useCallback(
+        (feature: mapboxgl.MapboxGeoJSONFeature, lngLat: mapboxgl.LngLatLike) => {
+            setClickedPointProperties({
+                feature: feature as unknown as ClickedPoint['feature'],
+                lngLat,
+            });
+            return false;
+        },
+        [],
+    );
+    const handlePointClose = useCallback(() => {
+        setClickedPointProperties(undefined);
+    }, []);
+
+    const handlePointClick = useCallback(
+        (feature: mapboxgl.MapboxGeoJSONFeature, lngLat: mapboxgl.LngLatLike) => {
+            setClickedPointProperties({
+                feature: feature as unknown as ClickedPoint['feature'],
+                lngLat,
+            });
+            return true;
+        },
+        [setClickedPointProperties],
+    );
+    const maxLearning = useMemo(() => (
+        getMaxLearning(learning)
+    ), [learning]);
+
+    return (
+        <Container
+            className={_cs(styles.learningMap, className)}
+            footerClassName={styles.footer}
+            footerContent={(
+                <div className={styles.learningsLegend}>
+                    <div className={styles.legendLabel}>{strings.learningLegendLabel}</div>
+                    <div className={styles.legendContent}>
+                        <div
+                            className={styles.learningGradient}
+                            style={{ background: `linear-gradient(90deg, ${LEARNINGS_LOW_COLOR}, ${LEARNINGS_HIGH_COLOR})` }}
+                        />
+                        <div className={styles.labelList}>
+                            <NumberOutput
+                                value={0}
+                            />
+                            <NumberOutput
+                                value={maxLearning}
+                            />
+                        </div>
+                    </div>
+                </div>
+            )}
+        >
+            <BaseMap
+                baseLayers={(
+                    <MapLayer
+                        layerKey="admin-0"
+                        hoverable
+                        layerOptions={adminFillLayerOptions}
+                        onClick={handleCountryClick}
+                    />
+                )}
+            >
```

Comment:
Please use GlobalMap instead of BaseMap.
We are deprecating the use of BaseMap in the future.